RELEASE.rst

```diff
@@ -0,0 +1,11 @@
+RELEASE_TYPE: patch
+
+This release makes two changes:
+
+* It makes the calculation of some of the metadata that Hypothesis uses for
+  shrinking occur lazily. This should speed up performance of test case
+  generation a bit because it no longer calculates information it doesn't need.
+* It improves the shrinker for certain classes of nested examples. e.g. when
+  shrinking lists of lists, the shrinker is now able to concatenate two
+  adjacent lists together into a single list. Shrinking may get somewhat slower
+  when the minimal example is large as a result of this change.
```

Comment:
This is an ambiguous clause - I think you mean "As a result of this change, shrinking may be slower when the minimal example is large".  If could however be read as "When the minimal example is large because of this change, shrinking may be slower".